src/xcode/ENA/ENA/Source/Services/Risk/Provider/Helper/CachedAppConfiguration.swift

```diff
@@ -55,19 +55,10 @@ final class CachedAppConfiguration {
 
 		self.deviceTimeCheck = deviceTimeCheck ?? DeviceTimeCheck(store: store)
 
-
 		guard shouldFetch() else { return }
 
 		// check for updated or fetch initial app configuration
-		getAppConfig(with: store.appConfigMetadata?.lastAppConfigETag)
-			.sink { response in
-				self.store.appConfigMetadata = AppConfigMetadata(
-					lastAppConfigETag: response.etag ?? "\"ReloadMe\"",
-					lastAppConfigFetch: Date(),
-					appConfig: response.config
-				)
-			}
-			.store(in: &subscriptions)
+		getAppConfig(with: store.appConfigMetadata?.lastAppConfigETag).sink(receiveValue: { _ in }).store(in: &subscriptions)
```

Comment:
Is this still needed if we don't do anything with the result?

Reply:
Yes, if the request succeeds we are saving it to the store.

Reply:
Inside of getAppConfig result handling

Reply:
Then maybe you could mark `getAppConfig()` with `@discardableResult` and remove `.sink(receiveValue: { _ in }).store(in: &subscriptions)` from this line?

Reply:
As far as I know, .sink needs to be called on the Future to execute the fetch.

Reply:
Makes sense 👍